.circleci/config.yml

```diff
@@ -52,89 +61,78 @@ commands:
             sudo cp /home/circleci/project/.circleci/docker-php-circleci.ini /usr/local/etc/php/conf.d/
             sudo apt-get update
             sudo apt-get install -y libpng-dev
-            sudo docker-php-ext-install gd
-  install_platformsh_tools:
-    description: "Install packages and tools for interacting with platform.sh API"
+            sudo docker-php-ext-install gd pcntl posix
+  hosts_keyscan:
+    description: "Keyscan for hosts that require SSH access"
     steps:
       - run:
-          name: Add extra OS and PHP extensions/config
-          command: |
-            sudo docker-php-ext-install pcntl posix
-      - run:
-          name: Keyscan Platform.sh region hostnames + GitHub for password-less access
+          name: Keyscan for hosts that require SSH access
           command: |
             mkdir -p ~/.ssh
-            ssh-keyscan -H ssh.$PLATFORM_REGION >> ~/.ssh/known_hosts
             ssh-keyscan -H github.com >> ~/.ssh/known_hosts
+            ssh-keyscan -H ssh.$PLATFORM_REGION >> ~/.ssh/known_hosts
+  install_psh_cli:
+    description: "Install the Platform.sh CLI tool"
+    steps:
       - run:
           name: Install the Platform.sh CLI tool
-          command: |
-            curl -sS https://platform.sh/cli/installer | php
-  github_hosts_workaround:
-    description: "Adds github.com to known hosts in container; for working locally."
-    steps:
+          command: curl -sS https://platform.sh/cli/installer | php
       - run:
-          name: Keyscan GitHub hostname
-          command: mkdir -p ~/.ssh && ssh-keyscan -H github.com >> ~/.ssh/known_hosts
+          name: Add platform cli tool to $PATH
+          command: echo 'export PATH="$HOME/"'.platformsh/bin':"$PATH"' >> $BASH_ENV
```

Comment:
Significantly simplifies using the platform CLI tool. Took me a while to understand how to get this stored for other things to use; a quirk of the CI service I guess.